python/tk_multi_workfiles/file_item.py

```diff
@@ -182,7 +182,19 @@ def step(self):
         """
         :returns:   The Shotgun step entity dictionary that this file is associated with.
         """
-        return self._details.get("step") or self._publish_details.get("step")
+        return (
```

Comment:
I don't think `any` would work here because we need the property to return the `step` value, but `any` just return True or False if there is any of the items given have a True boolean value.

Reply:
Oh I see. I thought only in a bool value instead. Thanks for clarifying.